NoClassDefFound for TypeArg

Using `smallrye-axle-web-client` I get the following error:

```
java.lang.NoClassDefFoundError: io/vertx/lang/axle/TypeArg
	at io.smallrye.restclient.reactive.ReactiveClientTest.badUrlFailure(ReactiveClientTest.java:50)
Caused by: java.lang.ClassNotFoundException: io.vertx.lang.axle.TypeArg
	at io.smallrye.restclient.reactive.ReactiveClientTest.badUrlFailure(ReactiveClientTest.java:50)
```

--- a/vertx-axle-clients/reactiverse-axle-postgres-client/pom.xml
+++ b/vertx-axle-clients/reactiverse-axle-postgres-client/pom.xml
@@ -33,15 +33,6 @@
             <version>${project.version}</version>
         </dependency>
 
-
-        <!-- Extra dependencies -->
-        <dependency>
-            <groupId>${project.groupId}</groupId>
-            <artifactId>smallrye-axle-generator</artifactId>
-            <version>${project.version}</version>
-            <scope>provided</scope>
-        </dependency>
-
         <dependency>
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>

--- a/vertx-axle-clients/vertx-axle-cassandra-client/pom.xml
+++ b/vertx-axle-clients/vertx-axle-cassandra-client/pom.xml
@@ -32,15 +32,6 @@
             <version>${project.version}</version>
         </dependency>
 
-
-        <!-- Extra dependencies -->
-        <dependency>
-            <groupId>${project.groupId}</groupId>
-            <artifactId>smallrye-axle-generator</artifactId>
-            <version>${project.version}</version>
-            <scope>provided</scope>
-        </dependency>
-
         <dependency>
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>

--- a/vertx-axle-clients/vertx-axle-consul-client/pom.xml
+++ b/vertx-axle-clients/vertx-axle-consul-client/pom.xml
@@ -32,15 +32,6 @@
             <version>${project.version}</version>
         </dependency>
 
-
-        <!-- Extra dependencies -->
-        <dependency>
-            <groupId>${project.groupId}</groupId>
-            <artifactId>smallrye-axle-generator</artifactId>
-            <version>${project.version}</version>
-            <scope>provided</scope>
-        </dependency>
-
         <dependency>
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>

--- a/vertx-axle-clients/vertx-axle-core/pom.xml
+++ b/vertx-axle-clients/vertx-axle-core/pom.xml
@@ -48,7 +48,6 @@
             <groupId>${project.groupId}</groupId>
             <artifactId>smallrye-axle-generator</artifactId>
             <version>${project.version}</version>
-            <scope>provided</scope>
         </dependency>
         <dependency>
             <groupId>${project.groupId}</groupId>

--- a/vertx-axle-clients/vertx-axle-kafka-client/pom.xml
+++ b/vertx-axle-clients/vertx-axle-kafka-client/pom.xml
@@ -58,14 +58,6 @@
         </dependency>
 
 
-        <!-- Extra dependencies -->
-        <dependency>
-            <groupId>io.smallrye.reactive</groupId>
-            <artifactId>smallrye-axle-generator</artifactId>
-            <version>${project.version}</version>
-            <scope>provided</scope>
-        </dependency>
-
         <dependency>
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>

--- a/vertx-axle-clients/vertx-axle-mail-client/pom.xml
+++ b/vertx-axle-clients/vertx-axle-mail-client/pom.xml
@@ -32,15 +32,6 @@
             <version>${project.version}</version>
         </dependency>
 
-
-        <!-- Extra dependencies -->
-        <dependency>
-            <groupId>${project.groupId}</groupId>
-            <artifactId>smallrye-axle-generator</artifactId>
-            <version>${project.version}</version>
-            <scope>provided</scope>
-        </dependency>
-
         <dependency>
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>

--- a/vertx-axle-clients/vertx-axle-mongo-client/pom.xml
+++ b/vertx-axle-clients/vertx-axle-mongo-client/pom.xml
@@ -32,15 +32,6 @@
             <version>${project.version}</version>
         </dependency>
 
-
-        <!-- Extra dependencies -->
-        <dependency>
-            <groupId>${project.groupId}</groupId>
-            <artifactId>smallrye-axle-generator</artifactId>
-            <version>${project.version}</version>
-            <scope>provided</scope>
-        </dependency>
-
         <dependency>
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>

--- a/vertx-axle-clients/vertx-axle-rabbitmq-client/pom.xml
+++ b/vertx-axle-clients/vertx-axle-rabbitmq-client/pom.xml
@@ -57,15 +57,6 @@
             <version>3.4.13</version>
         </dependency>
 
-
-        <!-- Extra dependencies -->
-        <dependency>
-            <groupId>io.smallrye.reactive</groupId>
-            <artifactId>smallrye-axle-generator</artifactId>
-            <version>${project.version}</version>
-            <scope>provided</scope>
-        </dependency>
-
         <dependency>
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>

--- a/vertx-axle-clients/vertx-axle-redis-client/pom.xml
+++ b/vertx-axle-clients/vertx-axle-redis-client/pom.xml
@@ -32,15 +32,6 @@
             <version>${project.version}</version>
         </dependency>
 
-
-        <!-- Extra dependencies -->
-        <dependency>
-            <groupId>${project.groupId}</groupId>
-            <artifactId>smallrye-axle-generator</artifactId>
-            <version>${project.version}</version>
-            <scope>provided</scope>
-        </dependency>
-
         <dependency>
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>

--- a/vertx-axle-clients/vertx-axle-web-client/pom.xml
+++ b/vertx-axle-clients/vertx-axle-web-client/pom.xml
@@ -37,15 +37,6 @@
             <version>${project.version}</version>
         </dependency>
 
-
-        <!-- Extra dependencies -->
-        <dependency>
-            <groupId>${project.groupId}</groupId>
-            <artifactId>smallrye-axle-generator</artifactId>
-            <version>${project.version}</version>
-            <scope>provided</scope>
-        </dependency>
-
         <dependency>
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>

